modules/gitops-dex-role-mappings.adoc

```diff
@@ -0,0 +1,19 @@
+// Module is included in the following assemblies:
+//
+// * configuring-sso-for-argo-cd-on-openshift
+
+[id="dex-role-mappings_{context}"]
+= Mapping users to specific roles
+
+You can set the role of a specific user as `role:admin` on SSO through OpenShift. The user should be in a group in which the `cluster-admin` role is added. Argo CD role will not map if the user has a direct `ClusterRoleBinding` role. A quick fix is to:
+
+. Create a group named `cluster-admins` group.
+. Add the user to the group.
+. Apply the cluster-admin `ClusterRole` to the group.
```

Comment:
Remember to use a colon, not a period, at the end of a command if it precedes a code snippet. 

`Apply the cluster-admin ClusterRole to the group:`